invenio_pages/administration/views/pages.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio administration pages view module."""
+from flask_babelex import lazy_gettext as _
+from invenio_administration.views.base import (
+    AdminResourceDetailView,
+    AdminResourceEditView,
+    AdminResourceListView,
+)
+
+
+class PageListView(AdminResourceListView):
+    """Configuration for pages list view."""
+
+    api_endpoint = "/pages"
+    name = "Pages"
+    resource_config = "pages_resource"
+    search_request_headers = {"Accept": "application/json"}
+    title = "Pages"
+    category = "Pages"
+    pid_path = "id"
+    icon = "file alternate outline"
+
+    display_search = True
+    display_delete = False
+    display_create = False
+    display_edit = True
+
+    item_field_list = {
+        "url": {"text": _("Url"), "order": 1, "width": 1},
+        "title": {"text": _("Title"), "order": 2, "width": 2},
+        "content": {"text": _("Content"), "order": 3, "width": 5, "truncate": 250},
+        "template_name": {"text": _("Template Name"), "order": 4, "width": 3},
```

Comment:
I think the content of the page and template name are not needed in the search view. It is overloading the user with information. In addition, the content is not really nicely displayed due to html format